.github/workflows/lint-test.yaml

```diff
@@ -11,19 +11,37 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.1.0
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      # Python is required because `ct lint` runs Yamale (https://github.com/23andMe/Yamale) and
+      # yamllint (https://github.com/adrienverge/yamllint) which require Python
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: unguiculus/chart-testing-action@master
```

Comment:
Now that this has passed CI, do you want to cut a new release for helm/chart-testing-action/releases and change this line to match?

Reply:
This should, of course, be `helm/chart-testing-action@master`.